google-cloud-core/src/main/java/com/google/cloud/Policy.java

```diff
@@ -236,11 +334,8 @@ public final Builder setEtag(String etag) {
       return this;
     }
 
-    /**
-     * Sets the version of the policy. The default version is 0, meaning only the "owner", "editor",
-     * and "viewer" roles are permitted. If the version is 1, you may also use other roles.
-     */
-    protected final Builder setVersion(int version) {
+    /** Sets the version of the policy. */
+    public final Builder setVersion(int version) {
       this.version = version;
```

Comment:
Looks like `Policy` itself could become an `@AutoValue` class, and you could drop the `equals`/`hashCode`/`toString` implementations.

Reply:
I'm hesitant to convert Policy class to AutoValue until a future pass. So holding off on this for now.